app/scripts/test-helpers/index.js

```diff
@@ -1,7 +1,5 @@
-// @ts-nocheck
 export {
   changeOptions,
-  waitForElements,
```

Comment:
@pkerpedjiev I think we should avoid exporting anything new out of this module that we don't use in tests. This gives us more flexibility to make changes in the future without having to go fix tests.